src/functionalTest/kotlin/com/vaadin/gradle/VaadinSmokeTest.kt

```diff
@@ -53,68 +56,99 @@ class VaadinSmokeTest : AbstractGradleTest() {
 
     @Test
     fun smoke() {
-        build("vaadinClean")
+        testProject.build("vaadinClean")
     }
 
     @Test
     fun testPrepareFrontend() {
-        build("vaadinPrepareFrontend")
+        testProject.build("vaadinPrepareFrontend")
 
-        val generatedFlowBuildInfoJson = File(testProjectDir, "build/vaadin-generated/META-INF/VAADIN/config/flow-build-info.json")
-        expect(true, generatedFlowBuildInfoJson.toString()) { generatedFlowBuildInfoJson.isFile }
+        val tokenFile = File(testProject.dir, "build/vaadin-generated/META-INF/VAADIN/config/flow-build-info.json")
+        expect(true, tokenFile.toString()) { tokenFile.isFile }
```

Comment:
I think it is a good idea to improve the `expect` messages everywhere. But, it is not critical for now to block the release.

Reply:
I agree - the tests were created hastily and refactored just recently. It would be good to revisit those expects() and fill in a proper error message.

Reply:
(resolved in master)